### ISSUE TITLE
Change Windows Environment Variable `HOME` to something else (like `QUCSHOME`)

### ISSUE THREAD

**Issue by jrast** [Qucs/qucs#204]
With the current setup, using `HOME` to define the Project and Settings directory, Qucs is causing problems as other programs might also look for a environment variable called `HOME`. In my case this is cygwin wich relies on the HOME Variable, but there might be other tools rely on this variable (which is also not nice from these tools to use generic variable like HOME). 

My recomondation ist to change this variable to something more specific like `QUCSHOME`.

**Comment by guitorri** [comment]
I think HOME is not used anymore and can be dropped from the Inno instaler script.

These are the environment variables that we currently use:
```
./qucs/main.cpp:641:  char* var = getenv("QUCSDIR");
./qucs/main.cpp:672:  var = getenv("ADMSXMLBINDIR");
./qucs/main.cpp:689:  var = getenv("ASCOBINDIR");
./qucs/main.cpp:706:  var = getenv("OCTAVEBINDIR");
```

**Comment by guitorri** [comment]
Should be fixed with #255
